README.md

```diff
@@ -844,7 +844,8 @@ Below is the complete list of available options that can be used to customize yo
 | `GITLAB_NOTIFY_PUSHER` | Add pusher to recipients list of broken build notification emails. Defaults to `false` |
 | `GITLAB_REPOS_DIR` | The git repositories folder in the container. Defaults to `/home/git/data/repositories` |
 | `GITLAB_BACKUP_DIR` | The backup folder in the container. Defaults to `/home/git/data/backups` |
-| `GITLAB_BACKUP_DIR_CHOWN` | Optionally change ownership of backup files on start-up | Defaults to `true` |
+| `GITLAB_BACKUP_DIR_CHOWN` | Optionally change ownership of backup files on start-up. Defaults to `true` |
+| `GITLAB_BACKUP_DIR_GROUP` | Optionally group backups into a subfolder. Can also be used to place backups in to a subfolder on remote storage. Not used by default. |
```

Comment:
I found it also here not clear what this variables means. I think a better name would be directly the reuse of this variable `GITLAB_BACKUP_DIRECTORY` or `GITLAB_BACKUP_TARGET_DIRECTORY`

Reply:
See https://docs.gitlab.com/ee/raketasks/backup_restore.html#specifying-a-custom-directory-for-backups